### PR TITLE
enable dynamic PVs in Kubernetes 1.11

### PR DESCRIPTION
**Pull request by BenTheElder**
follow up to https://github.com/kubernetes-sigs/kind/pull/1261

### CODE REVIEW

**Comment by BenTheElder** [comment]
thanks @tamalsaha 
/cc @aojea @amwat

File: pkg/cluster/internal/kubeadm/config.go [hunk 1]
```diff
@@ -134,6 +134,20 @@ nodeRegistration:
     node-ip: "{{ .NodeAddress }}"
 networking:
   podSubnet: "{{ .PodSubnet }}"
+# we need this to allow dynamic PVs to work properly
+apiServerExtraArgs:
+  "feature-gates": "DynamicProvisioningScheduling=true"
```
Comment (aojea) [inline pkg/cluster/internal/kubeadm/config.go]: the docs says it was superseded by `VolumeScheduling`
https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/

Comment (BenTheElder) [reply pkg/cluster/internal/kubeadm/config.go]: this config is for kubernetes v1.11 :upside_down_face:

Comment (BenTheElder) [reply pkg/cluster/internal/kubeadm/config.go]: it was deprecated in 1.12

Comment (aojea) [reply pkg/cluster/internal/kubeadm/config.go]: ahh, but only after 1.12 

> DynamicProvisioningScheduling: Extend the default scheduler to be aware of volume topology and handle PV provisioning. This feature is superseded by the VolumeScheduling feature completely in v1.12.

**Comment by tamalsaha** [comment]
Thanks a lot @BenTheElder ! I know you just cut a release for kind. If it is not too much trouble, can we get a release with these changes?

**Pull request merged**